cabal-testsuite/src/Test/Cabal/CheckArMetadata.hs

```diff
@@ -25,7 +25,7 @@ import Distribution.Package               (getHSLibraryName)
 import Distribution.Simple.Compiler       (compilerId)
 import Distribution.Simple.LocalBuildInfo (LocalBuildInfo, compiler, localUnitId)
 
--- Almost a copypasta of Distribution.Simple.Program.Ar.wipeMetadata
+-- Almost a copy/paste of Distribution.Simple.Program.Ar.wipeMetadata
```

Comment:
Revert. this is fine and is common internet vernacular.

Reply:
Personally, I'd prefer "... a copy and paste of ..."

Reply:
> Personally, I'd prefer "... a copy and paste of ..."

same with me

Reply:
But the question is not what you both prefer, it's whether or not the original text was clear in its intent and ostensible correctness, and *not touching unless necessary*. We want meaningful changes, not just change for the sake of it. This to me does not meet the bar for meaningful change.

Reply:
ok, fair enough. In the context of the PR, this change looked appropriate to me. Happy to reconsider if you feel strongly against it.